src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -190,6 +190,7 @@ public static class Miscellaneous {
         public @Option @Number(max = 60 * 12, step = 10) int REFRESH_TIME = 60;
         public @Option @Number(max = 60 * 12, step = 10) int PAUSE_FOR = 0;
         public @Option boolean RESET_REFRESH = true;
+        public @Option boolean SOLVE_BACKPAGE_CAPTCHA = false;
```

Comment:
this doesn't seem to be used at all in the PR, you sure you're not missing something?
You're detecting if captcha is a thing, but not ever solving it